utils/src/macros.rs

```diff
@@ -6,6 +6,14 @@ macro_rules! concat_slices {
     }
 }
 
+/// Concatenates provided byte slices and hashes result to a point on the curve. Returns as Affine coordinates.
+#[macro_export]
+macro_rules! affine_group_from_slices {
+    ($($arg: expr),+) => {
+        $crate::hashing_utils::affine_group_elem_from_try_and_incr::<_, D>(&$crate::concat_slices!($($arg),+))
```

Comment:
The counter should be in this maacro so that there is only 1 place where we convert it to bytes like this 
```
macro_rules! affine_group_from_slices {
  (label: expr, prefix: expr, counter: expr) => {
   
  }
}
```

and use it like `affine_group_from_slices!(label, b" : h_", i)`